### PR TITLE
fix #12365 [Accessibility] The text "Document does not contain any pages" in printPreviewControl has low color contrast

### LINKED ISSUE #12365: [Accessibility] The text "Document does not contain any pages" in printPreviewControl has low color contrast
### .NET version

.NET SDK 10.0.100-alpha.1.24510.13

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No, also repro in previous versions

### Issue description

The text "Document does not contain any pages" in printPreviewControl has low color contrast:
 Image

### Steps to reproduce

Repro steps:
1. Create a Winforms .NET application
2. Add printPreviewControl from toolBox to form designer or call printPreviewDialog to showDialog
3. Build and run
4. Use AccessibilityInsight tool to detect the text's color contrast in printPreviewControl

### PR DESCRIPTION
**Pull request by Epica3055**
Fixes #12365 

## Proposed changes

- 
- Change the text color to black
- 

## Regression? 

- No

## Risk

- low

### Before

 Image
### After

image

## Test methodology 

- 
- manual 
- 

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12368)

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewControl.cs [hunk 1]
```diff
@@ -592,7 +592,7 @@ private void CalculatePageInfo()
 
     private void DrawMessage(Graphics g, Rectangle rect, bool isExceptionPrinting)
     {
-        using var brush = ForeColor.GetCachedSolidBrushScope();
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewControl.cs:595]: Where are we setting forecolor to white? Does it work well when we have document to display?
Hardcoded color might not work in other color schemes, can we use ControlText color(or brush) instead?

Comment (Epica3055) [reply src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewControl.cs:595]: > Where are we setting forecolor to white?

https://github.com/dotnet/winforms/blob/1d963e9d2168a60d37159b967b1899e2540a98f1/src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewControl.cs#L56C1-L59C26


https://github.com/dotnet/winforms/blob/1d963e9d2168a60d37159b967b1899e2540a98f1/src/System.Windows.Forms/src/System/Windows/Forms/Printing/PrintPreviewControl.cs#L298

> Does it work well when we have document to display?

image

> Hardcoded color might not work in other color schemes, can we use ControlText color(or brush) instead?

Good idea 👍

**Review by LeafShi1 — COMMENTED** [review]
LGTM!

**Comment by Tanya-Solyanik** [comment]
Loogs good, but could you please get this change tested by @Olina-Zhang's team?

**Comment by LeafShi1** [comment]
1 non-regression issue was found in HC mode, and a new issue [12401](https://github.com/dotnet/winforms/issues/12401) has been filed to track it.

**Review by LeafShi1 — APPROVED** [review]

**Review by Tanya-Solyanik — APPROVED** [review]

**Comment by Tanya-Solyanik** [comment]
@Epica3055 - please prepare this one for servicing

**Pull request merged**